### ISSUE TITLE
Replace the taco message?

### ISSUE THREAD

**Issue by GBKS** [saving-satoshi/saving-satoshi#185]
It's fun, but decoding "tacos for everyone" in [chapter 1, challenge 2](https://savingsatoshi.com/en/chapters/chapter-1/transacting-2) has nothing to do with the story. Is there a more relevant message we could use?

**Comment by ecurrencyhodler** [comment]
Haha love that idea ben! Our plan is to actually create our own op_code that is essentially a clue into Ch. 2. 

We had discussed replacing this challenge with our own op_code but I think the last thing we brought up was to keep it for the demo and replace it for v1. We can revisit this topic on our next call.

**Comment by adamjonas** [comment]
@ecurrencyhodler I believe you mean op_return. And until then, I think we could find a better placeholder as I agree it feels quite out of place.

**Comment by ecurrencyhodler** [comment]
I recommend using this as a placeholder until we choose a final one even though it mentions CA as a state.
https://blockstream.info/tx/ee3b8caaeb58245338dd299467de89ec6833d2a4235493c95059934603b5e98d?expand

**Comment by adamjonas** [comment]
I don't know the history of this one but let's make sure it's not a reference to Dorian Nakamoto who is as being harassed for his last name around that time period.

**Comment by ecurrencyhodler** [comment]
I don't think it is. It's a fake street address and Dorian's house reportedly is in a different city. 

I have another one in vegas we could use as an alternative which also is a fake address but looks like has geographic coordinates.

**Comment by adamjonas** [comment]
Ok. If it's not then it's not a problem. Carry on.

**Comment by ecurrencyhodler** [comment]
Okay. I will move this to "To Do" and we will use this txn. 

https://blockstream.info/tx/ee3b8caaeb58245338dd299467de89ec6833d2a4235493c95059934603b5e98d?expand
